modules/network-observability-metrics.adoc

```diff
@@ -0,0 +1,54 @@
+// Module included in the following assemblies:
+//
+// network_observability/metrics-alerts-dashboards.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="network-observability-metrics_{context}"]
+= Network Observability metrics
+Metrics generated by the `flowlogs-pipeline` are configurable in the `spec.processor.metrics.includeList` of the `FlowCollector` custom resource to add or remove metrics. 
+
+You can also create alerts by using the `includeList` metrics in Prometheus rules, as shown in the example "Creating an alert for high ingress traffic".
+
+When looking for these metrics in Prometheus, such as in the Console through Observe -> Metrics, or when defining alerts, all the metrics names are prefixed with "netobserv_". For example, "netobserv_namespace_flows_total". Available metrics names are as follows.
+
+.includeList metrics names
+[options="header"]
+Names followed by an `*` are enabled by default. 
+
+* `namespace_egress_bytes_total`
+* `namespace_egress_packets_total`
+* `namespace_ingress_bytes_total`
+* `namespace_ingress_packets_total`
+* `namespace_flows_total` *
+* `node_egress_bytes_total`
+* `node_egress_packets_total`
+* `node_ingress_bytes_total` *
+* `node_ingress_packets_total`
+* `node_flows_total`
```

Comment:
This list isn't quite in alphabetical order in a few places, just checking if it should be